packages/build/src/error/type.js

```diff
@@ -152,12 +167,20 @@ const TYPES = {
 
   // `@netlify/build` threw an uncaught exception
   exception: {
+    showInBuildLog: true,
     title: 'Core internal error',
     stackType: 'stack',
     showErrorProps: true,
     rawStack: true,
     severity: 'error',
   },
+
+  // Errors related with the telemetry output
+  telemetry: {
+    showInBuildLog: false,
+    title: 'Telemetry error',
+    severity: 'success',
```

Comment:
What are your thoughts on making `severity` `error` instead since those are technically system errors? This would ensure those get reported as system errors in Bugsnag, and that we are notified for them as such.

When it comes to ensuring the `severityCode` is not modified by telemetry errors, this should already be handled by computing `severityCode` before telemetry starts.

Reply:
Overall I think it makes sense yeah. We'll most likely need to re-write our error handling in `main` though:

https://github.com/netlify/build/blob/7e4489982b17005f80c2b5c4b48df7ccb21cfeef/packages/build/src/core/main.js#L78-L81

As I believe an error from telemetry on build success would ovewrite our `severityCode` and exit the build with an error status 🤔

Reply:
We might need to rewrite the logic anyway, if we want to handle telemetry errors.
Namely, error handling should probably happen after telemetry, so it can capture telemetry errors.
So the flow would be: computing severity code, then telemetry, then error handling?

Reply:
I'm also thinking we could probably make adding this new type part of the separate follow up PR? That way we can merge this as it is (just the error handling changes, `showInBuildLog` field and the new comment at the top of this file)?

Reply:
That works :+1:

Reply:
Done via https://github.com/netlify/build/pull/2439/commits/d9654c345e222b5cb5589f37973a2907f90ce179